File: php/fizzbuzz.php
<?php

for ($i = 1; $i <= 30; $i++) {
  $m = [];
  if ($i % 3 === 0) {
      $m[] = 'fizz';
  }
  if ($i % 5 === 0) {
      $m[] = 'buzz';
  }
  print(($m ? implode('', $m) : $i) . PHP_EOL);
}

File: elixir/fizzbuzz.ex
Enum.each(1..30, fn v ->
  cond do
    rem(v, 15) == 0 -> IO.puts("fizzbuzz")
    rem(v, 3) == 0 -> IO.puts("fizz")
    rem(v, 5) == 0 -> IO.puts("buzz")
    true -> IO.puts(v)
  end
end)
